versions/l-/libassert.json

```
@@ -0,0 +1,10 @@
+{
+    "versions": [
+      {
+        "git-tree": "100847da8e340e7e4c05f1df76281eecf733d848",
+        "version": "1.2.1",
+        "port-version": 0
+      }
+    ]
+  }
+  
```

```suggestion
  }

```

Format: Do not indent the blank lines at the end of the text.